ifs/CMakeLists.txt

```diff
@@ -28,6 +28,9 @@ ecbuild_add_library(
     PRIVATE_LIBS
         ecrad.${PREC}
         ecrad_base.${PREC}
+        ecrad_utilities.${PREC}
```

Comment:
This new dependency is introduced to be able to use roctx in radiation_scheme & co later, I suppose?

Reply:
yes